foundation-ui/utils/ReactUtil.js

```diff
@@ -15,6 +15,10 @@ const ReactUil = {
       return null;
     }
 
+    if (Array.isArray(elements) && elements.length === 1 && !alwaysWrap) {
```

Comment:
Come to think of it, it sort adds some inconsistency, since we are allowing some types of elements here and only ReactElements in the next statement.
Talked with @jfurrow offline about it and we decided to add the check and if we start having the need for the other types of elements, let's change that in a separate PR.